apps/backlite/settings.js

```diff
@@ -1,19 +1,20 @@
 (function(back) {
   var FILE = "BackLite.settings.json";
-  // Load settings
-  var settings = Object.assign({
-      brightness: 0.3,
-  }, require('Storage').readJSON(FILE, true) || {});
+
+  // Load settings safely (avoid crashes on bad/missing JSON)
+  var settings = require("Storage").readJSON(FILE, 1) || {};
+  
+  if (!isFinite(settings.brightness)) settings.brightness = 0.3;
 
   function writeSettings() {
-    require('Storage').writeJSON(FILE, settings);
+    require("Storage").writeJSON(FILE, settings);
   }
 
-  // Show the menu
   E.showMenu({
     "" : { "title" : "BackLite" },
-    'Brightness': {
-      value: 0.3|settings.brightness,  
+    "< Back": () => (back()), // fallback if run standalone
```

Comment:
This can save a bit of memory with:

```suggestion
    "< Back": back, // fallback if run standalone
```